src/csvcubed/utils/sparql_handler/csvw_inspector.py

```diff
@@ -48,6 +54,26 @@ def csvw_type(self) -> CSVWType:
                 "The input metadata is invalid as it is not a data cube or a code list."
             )
 
+    @cached_property
+    def _table_schema_properties(self) -> TableSchemaPropertiesResult:
+        """
+        Cached property for the select_table_schema_properties query that stores the query's results.
+        """
+        result = select_table_schema_properties(self.rdf_graph)
```

Comment:
Shouldn't each csv inside the CSV-W have their own table schema properties? i.e. I think this should be a dictionary mapping `csv_url` to `TableSchemaPropertiesResult`.

Reply:
Done